src/Timestamps/ExtraSmallTimestampsHTMLFormatter.php

```diff
@@ -17,10 +17,10 @@ class ExtraSmallTimestampsHTMLFormatter implements TimestampsFormatterInterface
 
     private $fmtMonth;
 
-    public function __construct()
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.